tests/integration/nm/ovs_test.py

```diff
@@ -378,6 +378,32 @@ def ovs_bridge_over_bond_system_iface_with_same_name(eth1_up, eth2_up):
     time.sleep(1)
 
 
+@pytest.fixture
+def ovs_bridge_and_ovs_interface_with_same_name_ipv4():
+    cmdlib.exec_cmd(
+        "nmcli c add type ovs-port conn.interface br-ex master br-ex con-name "
```

Comment:
IMHO it is better style to use non-abbreviated names in scripts/documentation, such as "nmcli connection add ...."

Reply:
+1